src/crimson/os/cyan_store.cc

```diff
@@ -646,4 +646,9 @@ boost::intrusive_ptr<Collection> CyanStore::get_collection(const coll_t& cid)
   return cp->second;
 }
 
+unsigned CyanStore::get_max_attr_name_length() const
```

Comment:
@liu-chunmei please avoid using merge/pull, which will bring merge commit to a PR. instead i'd suggest use rebase next time. i will do rebase on behalf of you this time.